### PR TITLE
Add support of multiple methods

### PR DESCRIPTION
**Pull request by vchirikov**
Subj.
PR adds support of `.Verify` for Moq also contains small allocation fixes.

If you want more info: https://github.com/dotnet/roslyn/issues/5835

### CODE REVIEW

**Comment by vchirikov** [comment]
Maybe I made mistake in `NSubstitute` changes, check this please by yourself

**Comment by MrLuje** [comment]
Sorry for the delay, I'll have a look today

File: Mocking.Helpers/Mocking.Helpers.Vsix/source.extension.vsixmanifest [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="Mocking.Helpers.DC82BF19-0482-4D47-A1FF-7F4CBBAA44D4" Version="1.0.0.0" Language="en-US" Publisher="MrLuje"/>
+    <Identity Id="Mocking.Helpers.DC82BF19-0482-4D47-A1FF-7F4CBBAA44D4" Version="1.1.14.0" Language="en-US" Publisher="MrLuje"/>
```
Comment (MrLuje) [inline Mocking.Helpers/Mocking.Helpers.Vsix/source.extension.vsixmanifest]: Leave it to 1.0.0.0, the build on appveyor handles it

File: Mocking.Helpers/Mocking.Helpers/NSubstitute/NSubstituteArgAnyCompletion.cs [hunk 2]
```diff
@@ -42,6 +40,7 @@ public override async Task ProvideCompletionsAsync(CompletionContext context)
                 var mockedMethodArgumentList = token.Parent as ArgumentListSyntax;
                 var mockedMethodInvocation = mockedMethodArgumentList.Ancestors()
                                                                      .OfType<InvocationExpressionSyntax>()
+                                                                     .Where(IsSubstituteForMethod)
```
Comment (MrLuje) [inline Mocking.Helpers/Mocking.Helpers/NSubstitute/NSubstituteArgAnyCompletion.cs]: This one isn't needed, in NSubstitute there is no method to declare that you are setting up your mock
**For** isn't part of the actual expression whereas it is the case for Moq with **Setup**
```csharp
Substitute.For<Calculator>().Add(Arg.Is(0), Arg.Is(1))
calculatorMock.Setup(calc => calc.Add(0, 1))
```

The only issue I have by not properly checking it, is that I have autocomplete suggestions also for regular methods, not just Substitute mock, but it's not a big deal

**Comment by vchirikov** [comment]
Done

**Comment by MrLuje** [comment]
Thanks !

**Pull request merged**